edit/edit.go

```diff
@@ -265,7 +265,7 @@ func IndexOfRuleByName(f *build.File, name string) (int, *build.Rule) {
 
 		// Allow for precisely targeting the package declaration. This
 		// helps adding new load() and license() rules
-		if name == "!package" {
+		if name == "__pkg__" {
 			if rule, ok := ExprToRule(stmt, "package"); ok {
```

Comment:
What happens if there's no call to `package` in the file?
Could you call `PackageDeclaration()` instead?

Reply:
Calling PackageDeclaration() would create the rule if it doesn't exist, so that wouldn't be the function to use for targeted placement like what is being done here. It and its counterpart ExistingPackageDeclaration are working on the build.File level to return a rule, but I'm trying to get a line position. This ExprToRule usage is exactly what ExistingPackageDeclaration is doing under the hood.

If there's no call to package in the file, the insertions work exactly like they would if the package() call existed in the preferred location as described by the documentation (after loads, variables, etc. but before rules)